js/src/templates/map/targets.hbs

```diff
@@ -1,24 +1,10 @@
-<div class="m-dashboard-target">
-  <div class="header">
-    <button class="js--toggle-target btn-toggle-target">+</button>
-    <h2 class="target-title js--target" data-slug="significantly_reduce_violence">Target name</h2>
-  </div>
-
-  <div class="indicators-wrapper official-indicators">
-    <span class="indicator-type">official indicators</span>
-    <div class="indicator-wrapper">
-      <input type="radio" name="indicator" value="number_of_conflict_deaths" id="number_of_conflict_deaths">
-      <label for="number_of_conflict_deaths"><h1 class="indicator-title">indicator name</h1></label>
-      <button class="js--indicator-info" data-slug="indicator-slug">i</button>
-    </div>
-  </div>
-
-  <div class="indicators-wrapper shadow-indicators">
-    <span class="indicator-type">shadow indicators</span>
-    <div class="indicator-wrapper">
-      <input type="radio" name="indicator" value="percentage_of_people_violence">
-      <label for="percentage_of_people_violence"><h1 class="indicator-title">indicator name</h1></label>
-      <button class="js--indicator-info" data-slug="percentage_of_people_violence">i</button>
+{{#each targets}}
+  <div class="m-dashboard-target" id="{{this.slug}}">
+    <div class="header">
+      <button class="js--toggle-target btn-toggle-target" data-slug={{this.slug}}>+</button>
+      <input type="radio" name="layer-selector" value={{this.slug}} id={{this.slug}} class="js--layer-selector -target">
+      <label for={{this.slug}}><h2 class="target-title js--target" data-slug={{this.slug}}>{{this.title}}</h2></label>
```

Comment:
When you are in a Handlebars loop you don't to use "this" to access to the property, you can access directly. Replace  {{ this.title }}  to {{ title }} and so on.